src/Bundle/Command/StubCommand.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\Command;
```

Comment:
Perhaps we should adjust namespace to something like CLI to avoid name collision with command/events from CQRS?
```suggestion
namespace Sylius\Bundle\ResourceBundle\CLI;
```

Reply:
I am opiniated cause `Command` is the naming convention for console commands.